app/views/home/index.html.erb

```diff
@@ -1,7 +1,18 @@
-<p>Hello World!</p>
-
-<% if current_user.admin? %>
-  <%= link_to t("presences"), presences_path %>
-<% end %>
-
-<%= link_to t("schedules"), new_schedule_path %>
+<div class="Home-Wrapper">
+  <div class="Home-Header">
+    <h1><%= t("welcome") %> <%= current_user.username %></h1>
+    <div class="Home-Logout">
+      <% if current_user %>
+        <%= button_to logout_path, {class: 'Button'} do %>
```

Comment:
Don't forget to pad hashes with spaces `{ class: 'Button'  }`

Reply:
Bump. Also, Why single quotes? I made the example with single quotes but please use double quotes always xD Sorry

Reply:
Because my rubocop if not cooperating and I thought that was supposed to be the rule